src/linker/Linker.Steps/OutputStep.cs

```diff
@@ -38,6 +39,10 @@ namespace Mono.Linker.Steps {
 
 	public class OutputStep : BaseStep {
 
+		public static string AssemblyListFile {
```

Comment:
I would prefer a new property on LinkConext that is set by the Driver and then accessed here. That is the more established pattern